thriftpy2/parser/parser.py

```diff
@@ -62,9 +62,20 @@ def p_include(p):
     for include_dir in replace_include_dirs:
         path = os.path.join(include_dir, p[2])
         if os.path.exists(path):
-            child = parse(path)
-            setattr(thrift, child.__name__, child)
+            child_path = os.path.normpath(
+                os.path.dirname(str(thrift.__name__).replace("_thrift", "").replace(".", os.sep)) + os.sep + p[2])
+
+            child_path = child_path.lstrip(os.sep)
+
+            child_module_name = str(
+                child_path).replace(os.sep,
+                                    ".").replace(
+                ".thrift", "_thrift")
```

Comment:
These code's indents are a little bit strange, I think it's better to using something like:

```
aaa = bbb().ccc()\
    .ddd()
```